VisualStudio/Program.cs

```diff
@@ -65,16 +69,51 @@ public async Task<int> RunAsync()
             return 0;
         }
 
+        protected virtual void ShowVersion()
+        {
+            output.WriteLine($"{ThisAssembly.Metadata.AssemblyName} {ThisAssembly.Metadata.Version}");
+            output.WriteLine();
+        }
+
         protected virtual void ShowUsage()
         {
-            Console.WriteLine();
-            Console.WriteLine($"Usage: {ThisAssembly.Metadata.AssemblyName} [command] [options|-?|-h|--help] [--save=]");
-            Console.WriteLine();
-            Console.WriteLine("Supported commands:");
+            output.WriteLine();
+            output.WriteLine($"Usage: {ThisAssembly.Metadata.AssemblyName} [command] [options|-?|-h|--help] [--save=ALIAS[--global]]");
+            output.WriteLine();
+            output.WriteLine("Supported commands:");
 
             var maxWidth = commandFactory.GetRegisteredCommands().Select(x => x.Key.Length).Max() + 5;
             foreach (var command in commandFactory.GetRegisteredCommands())
-                Console.WriteLine($"  {command.Key.GetNormalizedString(maxWidth)}{command.Value.Description}");
+                output.WriteLine($"  {command.Key.GetNormalizedString(maxWidth)}{command.Value.Description}");
+        }
+
+        protected virtual void ShowExamples(string commandName)
+        {
+            var examplesFile = Path.Combine("Docs", commandName + ".md");
```

Comment:
Hm, so this implies that the docs folder will have to end up in the package somehow, for this to actually work... I was thinking embedding the resource might be easier that trying to get that packaging/unpackaging figured out...